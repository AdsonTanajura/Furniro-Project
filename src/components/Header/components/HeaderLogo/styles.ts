import styled from 'styled-components';

export const LogoContainer = styled.div`
  display: flex;
  justify-content: center;
  flex-wrap: wrap;
  align-items: center;
  gap: 5px;
`;

export const LogoTitle = styled.span`
  font-size: 34px;
  font-weight: bold;
  font-family: 'Montserrat', sans-serif;
`;
