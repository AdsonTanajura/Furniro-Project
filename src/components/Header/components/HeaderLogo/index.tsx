import React from 'react';
import { LogoContainer, LogoTitle } from './styles';
import Logo from '../../../../assets/Logo.svg';

const HeaderLogo = () => {
  return (
    <LogoContainer>
      <img src={Logo} alt="Logo" />
      <LogoTitle>Furniro</LogoTitle>
    </LogoContainer>
  );
};

export default HeaderLogo;
