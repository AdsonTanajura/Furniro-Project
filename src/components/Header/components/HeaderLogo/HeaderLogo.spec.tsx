import React from 'react';
import { render } from '@testing-library/react';
import HeaderLogo from '.';

describe('<HeaderLogo />', () => {
  it('should render Logo', () => {
    const { container, getByRole } = render(<HeaderLogo />);

    const logo = getByRole('img');

    expect(logo).toBeInTheDocument();
    expect(container.firstChild).toMatchSnapshot();
  });
});
