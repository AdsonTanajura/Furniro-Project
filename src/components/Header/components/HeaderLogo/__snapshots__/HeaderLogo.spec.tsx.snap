// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HeaderLogo /> should render Logo 1`] = `
.c0 {
  display: flex;
  justify-content: center;
  flex-wrap: wrap;
  align-items: center;
  gap: 5px;
}

.c1 {
  font-size: 34px;
  font-weight: bold;
  font-family: 'Montserrat',sans-serif;
}

<div
  class="c0"
>
  <img
    alt="Logo"
    src="test-file-stub"
  />
  <span
    class="c1"
  >
    Furniro
  </span>
</div>
`;
